src/js/tech/html5.js

```diff
@@ -767,7 +770,11 @@ class Html5 extends Tech {
       return super.removeRemoteTextTrack(track);
     }
 
-    var tracks, i;
+    let tracks, i;
+
+    // TODO: track can be html text element due to addRemoteTextTrack—consider refactoring
+    let trackElement = this.remoteTextTrackEls().getTrackElementByTrack_(track);
```

Comment:
Hopefully, with this update, this should only ever be an official `HTMLTrackElement` or our emulated implementation and not `{track: TextTrack}`